envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -117,4 +117,10 @@ class AuthorizationProperties {
 class EgressProperties {
     var clusterNotFoundStatusCode = 503
     var handleInternalRedirect = false
+    var http2Properties = Http2Properties()
```

Comment:
What do you think about renaming this to just http2? Class should stay `Http2Properties` but field might looks better with name http2.